script/security-check.js

```diff
@@ -11,6 +11,7 @@ const packageJSON = require('../package.json');
 const exceptionSet = new Set([
   'https://npmjs.com/advisories/996',
   'https://npmjs.com/advisories/1488',
+  'https://npmjs.com/advisories/1547',
```

Comment:
Adds an exception for the error we're seeing, http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/schemaorg-ldjson-faqpage/2/pipeline